packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -24,10 +24,12 @@
 		outline: $border-width solid transparent;
 		transition: 0.2s outline;
 		@include reduce-motion("transition");
-		resize: none;
 		margin-top: $default-block-margin;
 		margin-bottom: $default-block-margin;
 
+		// This needs high specificity as it's output as an inline style by the library.
```

Comment:
which library can override this?

Reply:
I also wonder why we had  the issue only in the site editor? Is it about the iframe? Does it mean we have another similar style elsewhere that we could remove now?

Reply:
I assume it's caused by `TextareaAutosize` which is used in context of the component. `resize: horizontal;` gets output as an inline style.

Reply:
Do you know why it doesn't happen in the post editor?

Reply:
I don't know why, for whatever reason the `resize` style does not appear to get output there. Here's some example markup:

```
<textarea role="button" aria-label="Add block" class="block-editor-default-block-appender__content" readonly="" rows="1" style="overflow: hidden; overflow-wrap: break-word; height: 25px;">Start writing or type / to choose a block</textarea>
```

Reply:
weird. Maybe the library detects an iframe or something and add it. Weird though.

Reply:
Agreed